Extension/package.json

```diff
@@ -2300,6 +2300,20 @@
         "./bin/cpptools-srv"
       ]
     },
+    {
+      "description": "C/C++ language components (Linux / aarch64)",
+      "url": "https://go.microsoft.com/fwlink/?linkid=2135628",
```

Comment:
These links will be updated for the 64-bit versions later, although the 32-bit binaries still work on the 64-bit OS Raspberry Pi OS (not sure if other OS's would have issues).